src/oasis-nexus/generated/api.ts

```diff
@@ -659,8 +659,6 @@ Currently only ERC-721 is supported, where the document is an Asset Metadata fro
   metadata_uri?: string;
   /** Identifies the asset which this NFT represents */
   name?: string;
-  /** The total number of transfers of this NFT instance. */
-  num_transfers?: number;
```

Comment:
this was hardcoded on FE side and we want to keep it. https://github.com/oasisprotocol/nexus/pull/585